spec/features/posts_spec.rb

```diff
@@ -0,0 +1,88 @@
+require 'rails_helper'
+RSpec.describe 'Posts' do
+  describe 'Posts #index' do
+    before do
+      visit user_posts_path(User.first)
```

Comment:
- [ ] For the project requirements, kindly ensure that all your tests are passing as according to the result image below, only two of them are passing.
![Screenshot from 2023-05-24 00-41-24](https://github.com/anderCM/blog-app/assets/83042742/07889833-12d9-407b-9203-9e7b3bdd0caa)

Reply:
Hi @tresorsawasawa  could you share with us extra information about the error messages? Because all we ran the tests and all of them are passing

![image](https://github.com/anderCM/blog-app/assets/77707836/ffa398a0-3906-4296-87ad-1d370b8043f5)